aixtend/modules/model.py

```diff
@@ -48,33 +47,22 @@ def __init__(
             id (str): ID of the Model
             name (str): Name of the Model
             supplier (str): supplier of the Model
-            api_key (str, optional): API key of the Model. Defaults to None.
+            api_key (str): Team API Key to run the model.
```

Comment:
What is the behaviour when?
1.  Team key is not present
2.  Team key is present and wrong
We just need to test these out.